### DISCUSSION TITLE
Is possible to retry commands that are placed in the before/beforeEach?

### DISCUSSION THREAD

**Discussion by melibe23** [cypress-io/cypress#17427] [Questions and Help]
I am having this issue https://github.com/cypress-io/cypress/issues/7062

I want cypress to try again when it happens. But the visit or request command are placed in a hook.

Is possible to retry commands that ara placed in the before/beforeEach?

**Comment by jennifer-shehane** [comment]
Cypress’s built-in retry-ability (queries + assertions) works the same inside before/beforeEach as in a test, cy.get(...).should(...) will retry there too. What doesn’t retry is the hook as a whole; if a hook fails, the spec’s tests are skipped. For flaky setup, add explicit assertions to wait on state, or enable test retries (which re-run beforeEach)
